css/theme/template/exposer.scss

```diff
@@ -0,0 +1,27 @@
+// Exposes theme's variables for easy re-use in CSS for plugin authors
+
+:root {
+  @include css-var(--background-color, $backgroundColor);
+  @include css-var(--main-font, $mainFont);
+  @include css-var(--main-font-size, $mainFontSize);
+  @include css-var(--main-color, $mainColor);
+  @include css-var(--block-margin, $blockMargin);
+  @include css-var(--heading-margin, $headingMargin);
+  @include css-var(--heading-font, $headingFont);
+  @include css-var(--heading-color, $headingColor);
+  @include css-var(--heading-line-height, $headingLineHeight);
+  @include css-var(--heading-letter-spacing, $headingLetterSpacing);
+  @include css-var(--heading-text-transform, $headingTextTransform);
+  @include css-var(--heading-text-shadow, $headingTextShadow);
+  @include css-var(--heading-font-weight, $headingFontWeight);
+  @include css-var(--heading1-text-shadow, $heading1TextShadow);
+  @include css-var(--heading1-size, $heading1Size);
+  @include css-var(--heading2-size, $heading2Size);
+  @include css-var(--heading3-size, $heading3Size);
+  @include css-var(--heading4-size, $heading4Size);
+  @include css-var(--code-font, $codeFont);
+  @include css-var(--link-color, $linkColor);
+  @include css-var(--link-color-hover, $linkColorHover);
+  @include css-var(--selection-background-color, $selectionBackgroundColor);
+  @include css-var(--selection-color, $selectionColor);
```

Comment:
node-sass is having trouble with this for me:

> no mixin named css-var

Do you want to use

```scss
  --background-color: #{$backgroundColor};
  --main-font: #{$mainFont};
  --main-font-size: #{$mainFontSize};
  --main-color: #{$mainColor};
  --block-margin: #{$blockMargin};
  --heading-margin: #{$headingMargin};
  --heading-font: #{$headingFont};
  --heading-color: #{$headingColor};
  --heading-line-height: #{$headingLineHeight};
  --heading-letter-spacing: #{$headingLetterSpacing};
  --heading-text-transform: #{$headingTextTransform};
  --heading-text-shadow: #{$headingTextShadow};
  --heading-font-weight: #{$headingFontWeight};
  --heading1-text-shadow: #{$heading1TextShadow};
  --heading1-size: #{$heading1Size};
  --heading2-size: #{$heading2Size};
  --heading3-size: #{$heading3Size};
  --heading4-size: #{$heading4Size};
  --code-font: #{$codeFont};
  --link-color: #{$linkColor};
  --link-color-hover: #{$linkColorHover};
  --selection-background-color: #{$selectionBackgroundColor};
  --selection-color: #{$selectionColor};
```

instead?

Reply:
I can absolutely do that yes. 
Don't remember how I stumbled upon the syntax in my PR, it was the only thing that worked on another project of mine (and as usual I forgot to document the why :laughing: ). 

I'll get a look at this today if I can.

Reply:
Also, it is a superset of another proposed change. I'll link the PR here too so we can close both.

Reply:
Which you found and closed :+1: